library/build.gradle

```diff
@@ -29,6 +29,8 @@ dependencies {
     compile('com.android.support.test.espresso:espresso-contrib:2.0') {
         exclude module: 'support-v4'
     }
+    compile 'com.android.support.test.uiautomator:uiautomator-v18:2.1.2'
+    compile 'com.android.support:appcompat-v7:25.1.0'
```

Comment:
Is this dependency a must? It can cause dependency conflicts on users of the library. That's why we added the exclude above...